src/core/transport/chttp2/stream_encoder.c

```diff
@@ -43,7 +43,7 @@
 #include "src/core/transport/chttp2/timeout_encoding.h"
 #include "src/core/transport/chttp2/varint.h"
 
-#define HASH_FRAGMENT_1(x) ((x) & 255)
+#define HASH_FRAGMENT_1(x) ((x)&255)
```

Comment:
This does not make any sense... We should fix whatever template we use for clang-format.